source/core.py

```diff
@@ -519,7 +520,8 @@ def handlePowerStatusChange(self):
 		except:
 			log.error("", exc_info=True)
 		if globalVars.appArgs.launcher:
-			gui.LauncherDialog.run()
+			from gui.startupDialogs import LauncherDialog
+			LauncherDialog.run()

```

Comment:
See earlier comment

Reply:
Same as above - isn't it better to import conditionally in these cases?